### DISCUSSION TITLE
Skiovox failing on last step

### DISCUSSION THREAD

**Discussion by Winterfyre** [3kh0/ext-remover#919] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
So I've been using Skiovox without issue for a while now, but for some reason, today when I accidentally exited out of the Skiovox account, I can't seem to get back in. The process works like normal up until the last step, where the Multiple Sign-in is Disabled popup appears, where now hitting escape doesn't close the popup. I've tried multiple times, it's a consistent issue.

(I thought I was CAUB'd, guess not, whoops)

**Comment by CGRmags** [comment]
@Winterfyre Dang I need a answer to this to because I still am in Skiovox so lol I need to know. :0

**Comment by S-PScripts** [comment]
use a different exploit or downgrade
